src/ast/passes/resolve_imports.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <sstream>
+#include <unordered_set>
+
+#include "ast/passes/resolve_imports.h"
+#include "ast/visitor.h"
+#include "bpftrace.h"
+#include "log.h"
+
+namespace bpftrace::ast {
+
+// Load imports emit unprocessed source, objects, etc.
```

Comment:
"Load imports then emit unprocessed source, objects, etc." ?

Reply:
I'll just drop this given that it's just a skeleton.